addons/mail/static/src/core/common/message.js

```diff
@@ -211,7 +203,7 @@ export class Message extends Component {
             "px-2": this.props.isInChatWindow,
             "opacity-50": this.props.thread?.composer.replyToMessage?.notEq(this.props.message),
             "o-actionMenuMobileOpen": this.state.actionMenuMobileOpen,
-            "o-editing": this.state.isEditing,
+            "o-editing": this.props.message.composer,
```

Comment:
It's obscure to use the composer directly as a means to determine if the message is in editing mode or not. 
I get the idea but I suggest a simple `message.isEditing` (or something else) to fill this role instead.

Reply:
This is an unnecessary indirection, or another point of failure (e.g. people adding custom things in `isEditing` and it becomes not the same as `message.composer`.

Reply:
I disagree; I prefer easy to read code instead of wondering what the composer has to do with the editing state.
But that's not really important